pkg/xray/resource_xray_security_policy.go

```diff
@@ -20,69 +21,84 @@ func resourceXraySecurityPolicyV2() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Name of the policy (must be unique)",
 			},
 			"description": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "More verbose description of the policy",
 			},
 			"type": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Type of the policy",
 			},
-			// not in create policy body, but it is in the get call response.
 			"author": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
```

Comment:
You know, if you have shared attributes between the 2 policies (which you do), then you can abstract them  and use the `mergeMap` function - this would reduce replication